src/server/scripts/Spells/spell_item.cpp

```diff
@@ -4345,7 +4337,7 @@ class spell_item_amalgams_seventh_spine : public AuraScript
             if (target->GetPrimarySpecialization() != spec)
                 target->RemoveAurasDueToSpell(aura);
             else if (!target->HasAura(aura))
-                target->CastSpell(target, aura, aurEff);
+                target->CastSpell(target, aura, GetEffect(EFFECT_0));
```

Comment:
Would we not need to validate this effect since OnHeartbeat is not really hooking any and implicitly validating any?